guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -95,23 +95,44 @@ Environment=GSS_USE_PROXY=1
 ----
 # systemctl restart httpd
 ----
-. Verify that SSO is working as expected.
-+
+
+IMPORTANT
+====
+With direct AD integration, HBAC through {FreeIPA} is not available.
+As an alternative, you can use Group Policy Objects (GPO) that enable administrators to centrally manage policies in AD environments.
+To ensure correct GPO to PAM service mapping, add the following sssd configuration to `/etc/sssd/sssd.conf`:
```

Comment:
```suggestion
To ensure correct GPO to PAM service mapping, add the following SSSD configuration to `/etc/sssd/sssd.conf`:
```

Reply:
Should this be capitalized in this instance since it's an acronym for System Security Services Daemon?

Reply:
+1 to @bangelic

Either capitalization or markup to denote a service name (`sssd`) would be required here. I'd personally prefer capitalization.

Reply:
Thanks, fixed!